src/storage/mod.rs

```diff
@@ -1943,7 +1961,30 @@ impl<E: Engine, L: LockManager, Api: APIVersion> Storage<E, L, Api> {
         Ok(())
     }
 
+    fn raw_batch_delete_requests_to_modifies(cf: CfName, keys: Vec<Vec<u8>>) -> Vec<Modify> {
+        keys.into_iter()
+            .map(|k| match Api::TAG {
+                ApiVersion::V1 | ApiVersion::V1ttl => {
+                    Modify::Delete(cf, Api::encode_raw_key_owned(k, None))
+                }
+                ApiVersion::V2 => {
+                    let raw_value = RawValue {
+                        user_value: vec![],
+                        expire_ts: None,
+                        is_delete: true,
+                    };
+                    Modify::Put(
```

Comment:
Can they be unified?

Reply:
En...add a method, e.g. "generate_delete_modify(keys: Vec<Vec<u8>>)", to trait [`APIVersion`](https://github.com/tikv/tikv/blob/45fcb0812a0437cabce2436c5223c96c9de83d5e/components/api_version/src/lib.rs#L15) ?

Reply:
Or maybe a method in this type to convert delete to Modify

Reply:
Fixed, PTAL~